### PR TITLE
e2e-metal-ipi dev-scripts job for 4.4

### PR DESCRIPTION
**Pull request by akiselev1**

### CODE REVIEW

File: ci-operator/jobs/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-release-4.4-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
+presubmits:
+  openshift-metal3/dev-scripts:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - release-4.4
```
Comment (andfasano) [inline ci-operator/jobs/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-release-4.4-presubmits.yaml:6]: This branch does not exist in the `dev-scripts` repo

File: ci-operator/jobs/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-release-4.4-postsubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+postsubmits:
+  openshift-metal3/dev-scripts:
+  - agent: kubernetes
+    branches:
+    - ^release-4\.4$
```
Comment (andfasano) [inline ci-operator/jobs/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-release-4.4-postsubmits.yaml:5]: Also this one

File: ci-operator/jobs/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-release-4.4-presubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+          - configMap:
+              name: cluster-profile-packet
+      - name: pull-secret
+        secret:
+          secretName: regcred
+    trigger: (?m)^/test( | .* )e2e-metal-ipi,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - release-4.4
```
Comment (andfasano) [inline ci-operator/jobs/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-release-4.4-presubmits.yaml:77]: And this one

**Comment by akiselev1** [comment]
Closing as superseded by https://github.com/openshift/release/pull/8934

**Pull request closed without merging**